src/components/Progress.js

```diff
@@ -0,0 +1,36 @@
+import React from "react"
+import { observer } from "mobx-react"
+
+import { Button, Link, List, InlineBlock, Heading, Provider, Divider } from "reakit";
+import theme from "reakit-theme-default";
+//import { blue } from "../colors"
+
+import CircularProgressbar from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
+
+import styled from "styled-components"
+import Layout from "../layouts/Text"
+
+
+// Need to make changes here
+const percentage = 25;
+
+const Progress = observer(({store}) => (
+  <Layout>
+        <Provider theme={theme}>
+            <div>
+                <Heading use="h4">
+                Progress
+                </Heading>
+                <CircularProgressbar
+                percentage={percentage}
+                text={`${percentage}%`} />
+            </div>
+        </Provider>
+  </Layout>
```

Comment:
Let's use two spaces for each level of indentation - can you clean this up?

I think there are VS Code syntax plugins that let you configure indents.

Reply:
Done, I'll try and be more consistent in the future.